cdk/stateful-api-stack.ts

```diff
@@ -0,0 +1,58 @@
+import { Construct, Stack, StackProps } from "@aws-cdk/core";
+import { Domain, ElasticsearchVersion } from '@aws-cdk/aws-elasticsearch';
+import { AnyPrincipal, Effect, PolicyStatement } from "@aws-cdk/aws-iam";
+import { BastionHostLinux, Vpc } from '@aws-cdk/aws-ec2';
+
+interface StatefulApiStackProps extends StackProps {
+  stage: string
+  esNodeType?: string
+}
+
+export default class StatefulApiStack extends Stack {
```

Comment:
In the future this is the stack that would deploy either Virtuoso or Neptune (depending on what we pick), right?